File: Sesion-07/Ejemplo-01/AppObjC/AppObjC/ViewController.m
//
//  ViewController.m
//  AppObjC
//
//  Copyright © 2019 Bedu. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
  [super viewDidLoad];
  // Do any additional setup after loading the view.
}


- (IBAction)showMessage:(id)sender {
  _label.text = [[NSString alloc] initWithString: @"Hello world"];
}
@end

File: Sesion-05/Ejemplo-02/ViewControllerLifeCycle/ViewControllerLifeCycle/ViewController.swift
//
//  ViewController.swift
//  ViewControllerLifeCycle
//
//  Copyright © 2019 Bedu. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

  override func viewDidLoad() {
    super.viewDidLoad()
    // Do any additional setup after loading the view.
  }

  override func viewWillAppear(_ animated: Bool) {
    super.viewWillAppear(animated)
    print(#function)
  }
  
  override func viewDidAppear(_ animated: Bool) {
    super.viewDidAppear(animated)
    print(#function)

  }
  
  override func viewWillDisappear(_ animated: Bool) {
    super.viewWillDisappear(animated)
    print(#function)
  }
}

